### PR TITLE
Update botkit 4.6.2

### PR DESCRIPTION
**Pull request by ihorrusinko**

### CODE REVIEW

File: lib/slack/ApiMock.js [hunk 1]
```diff
@@ -23,10 +23,22 @@ module.exports.bindMockApi = function (controller) {
...
 
-    /*
-    * monkey patch of axios mock adapter
-    * axios mock does not return request and headers objects for response
-    * */
+	/*
+	* workaround to prevent fail for slack api initialization
+	* https://github.com/howdyai/botkit/blob/v4.6.1/packages/botbuilder-adapter-slack/src/slack_adapter.ts#L128-L137
+	* */
+	process.exit = () => {};
```
Comment (amplicity) [inline lib/slack/ApiMock.js]: What's going on here?

Comment (amplicity) [reply lib/slack/ApiMock.js]: @ihorrusinko and I talked about this. he's doing it because botkit tries to ping slack api immediately in the constructor so we cannot mock slack api before the constructor is built. Is there a better way we could do this?

Comment (ihorrusinko) [reply lib/slack/ApiMock.js]: I've removed it. 
We use full botkit initialisation instead only bot version
https://github.com/gratifyguy/botkit-mock/pull/57/files#diff-08a4a90fec7ec7a039fb74fba5054ad2R82-R92

it also simplified initialisation of slack adapter  
https://github.com/gratifyguy/botkit-mock/pull/57/files#diff-e8bf6f445c0d957ef2fc5a81bb1851cdR15

File: lib/slack/ApiMock.js [hunk 2]
```diff
@@ -53,7 +58,6 @@ module.exports.bindMockApi = function (controller) {
     controller.axiosMockAdapter = axiosMockAdapter;
     controller.apiLogByKey = apiLogByKey;
     controller.httpBodyLog = httpBodyLog;
-    controller.adapter.slack.axios = axios;
```
Comment (amplicity) [inline lib/slack/ApiMock.js]: I see we configure some axios logic above but you removed this line. It looks like we still use axios below.

1. Why was this line removed?
2. Can you confirm we still require axios in this file?

File: lib/slack/ApiMock.js (hunk 1 above)
Comment (amplicity) [reply lib/slack/ApiMock.js]: Awesome work here Ihor.

File: lib/slack/ApiMock.js (hunk 2 above)
Comment (ihorrusinko) [reply lib/slack/ApiMock.js]: yes we still use it, you can see below in `spawn`
